test/kubemark/run-e2e-tests.sh

```diff
@@ -23,6 +23,7 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/../..
 ABSOLUTE_ROOT=$(readlink -f "${KUBE_ROOT}")
 
 source "${KUBE_ROOT}/cluster/kubemark/util.sh"
+source "${KUBE_ROOT}/cluster/skeleton/util.sh"
 
 echo "Kubemark master name: ${MASTER_NAME}"
 
```

Comment:
How about just outputting KUBE_MASTER_IP here and removing detect-master call instead of sourcing skeleton's util.sh because detect-master just outputs the IP and name of master and the name is already output at line 28.
```
# Must ensure that the following ENV vars are set
function detect-master {
        echo "KUBE_MASTER_IP: ${KUBE_MASTER_IP:-}" 1>&2
        echo "KUBE_MASTER: ${KUBE_MASTER:-}" 1>&2
}
```
/cc @oomichi

Reply:
Updated :)